package-lock.json

```diff
@@ -9090,10 +9090,21 @@
       }
     },
     "cypress-file-upload": {
-      "version": "3.5.3",
-      "resolved": "https://registry.npmjs.org/cypress-file-upload/-/cypress-file-upload-3.5.3.tgz",
-      "integrity": "sha512-S/czzqAj1BYz6Xxnfpx2aSc6hXsj76fd8/iuycJ2RxoxCcQMliw8eQV0ugzVlkzr1GD5dKGviNFGYqv3nRJ+Tg==",
-      "dev": true
+      "version": "4.0.4",
+      "resolved": "https://registry.npmjs.org/cypress-file-upload/-/cypress-file-upload-4.0.4.tgz",
+      "integrity": "sha512-q04HoVb8OQTpusHOadGER+9Dvif7kfoM8MjVAF37O+X5VEZlUARbXrKECH6pUH0vKmWRGcrsoB6mKah9ZJa4Pw==",
+      "dev": true,
+      "requires": {
+        "mime": "^2.4.4"
+      },
+      "dependencies": {
+        "mime": {
```

Comment:
This is why we can remove so much boilerplate: `cypress-file-upload` now requires `mime` which will allows it to do mime-type sniffing vs. us defining it manually.